### PR TITLE
[enh] adds a start action

### PR DESCRIPTION
**Pull request by albancrommer**

### CODE REVIEW

File: src/generator-vulcanjs/generators/start/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const chalk = require('chalk');
+const VulcanGenerator = require('../../lib/VulcanGenerator');
+
+module.exports = class extends VulcanGenerator {
+
+  default() {
+    this.log(chalk.green('\nStarting your app... \n'));
+    this.spawnCommandSync('meteor', ['run']);
```
Comment (mechanical-turk) [inline src/generator-vulcanjs/generators/start/index.js:8]: is there a reason you prefer `meteor run` instead of `meteor start` here?

**Review by mechanical-turk — COMMENTED** [review]
good job. just one comment, which you can see below

**Comment by SachaG** [comment]
Nice!

File: src/generator-vulcanjs/generators/start/index.js (hunk 1 above)
Comment (albancrommer) [reply src/generator-vulcanjs/generators/start/index.js:8]: Went there, did that ;)

`'start' is not a Meteor command. See 'meteor --help'.`

Or maybe it is a command now and my local install is outdated?

Comment (albancrommer) [reply src/generator-vulcanjs/generators/start/index.js:8]: My only concern was about using the various assert / errors or not. 

I did not use them, so you're not forced to use the scaffolding tool, `start` just does the job.

Comment (mechanical-turk) [reply src/generator-vulcanjs/generators/start/index.js:8]: Hmm im not convinced. What's your meteor version? 

My reasoning is that if we're doing a replacement for "mereor run", then the term should be "vulcan run".  But we're doing "vulcan start", so it should replace "meteor start". 

What do yoı think,

Comment (albancrommer) [reply src/generator-vulcanjs/generators/start/index.js:8]: Meteor 1.5.1 

I disagree, as the whole point is to offer to users an uniform command, which "hides" the background operations .

* It executes `meteor run`? vulcan start.
* It executes `yarn start --foo bar --dport 3200 --nocomply` ? Still vulcan start
* Now it spawns a docker ? Again, vulcan start 

If you insist, we can allow vulan run|start as an alias ? :)

**Pull request merged**